README.md

```diff
@@ -1 +1,72 @@
-# trip-agency
\ No newline at end of file
+# bookingTripRequest-agency
```

Comment:
Update ?

Btw, we should rename the repo and the artifact to something with `-agent` to be consistent with all other samples. I think it's fine to use `trip-agent`. I can rename the repo after merging this.